src/ontology/uberon.Makefile

```diff
@@ -24,6 +24,12 @@ all: uberon-qc
 	echo "make $@ succeeded..."
 
 
+# Enable second expansion of prerequisites (GNU Make only!).
+# We need this to expand the COLLECTED_*_SOURCES variables in the
+# composite pipeline.
+.SECONDEXPANSION:
```

Comment:
Are we sure that this does not at all affect anything in the normal, first expansion?

Reply:
Yes. All it does is allowing the use of this:

```Make
$(TMPDIR)/collected-%.owl: $$(COLLECTED_$$*_SOURCES)
```

That is, to define the prerequisites of a pattern rule using a variable whose name is itself dependent on the pattern.

Reply:
I remember in the past some strange issues with updating Variables like using the += operator and similar, but I guess we don't use such patterns.